src/site/stages/build/process-cms-exports/schemas/transformed/node-person_profile.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  type: 'object',
+  properties: {
+    contentModelType: { enum: ['node-person_profile'] },
+    entityType: { enum: ['node'] },
+    entityBundle: { enum: ['person_profile'] },
+    entityUrl: {
+      // This one doesn't have a breadcrumb, interestingly enough
+      type: 'object',
+      properties: {
+        path: { type: ['string', 'null'] },
+      },
+      required: ['path'],
+    },
+    fieldBody: { type: ['string', 'null'] },
+    fieldDescription: { type: ['string', 'null'] },
+    fieldEmailAddress: { type: ['string', 'null'] },
+    fieldLastName: { type: 'string' },
+    fieldMedia: { type: ['object', 'null'] },
```

Comment:
I had a hard time finding a `person_profile` with any real `field_media`, so I'm not sure what this would look like if it were there.

Reply:
I considered something like this, but for the moment our `Media` ref takes care of the possibility that it can be null: 

```diff
--- a/src/site/stages/build/process-cms-exports/schemas/transformed/node-person_profile.js
+++ b/src/site/stages/build/process-cms-exports/schemas/transformed/node-person_profile.js
@@ -16,7 +16,9 @@ module.exports = {
     fieldDescription: { type: ['string', 'null'] },
     fieldEmailAddress: { type: ['string', 'null'] },
     fieldLastName: { type: 'string' },
-    fieldMedia: { type: ['object', 'null'] },
+    fieldMedia: {
+     oneOf: [{ $ref: 'Media' }, { type: 'null' }],
+    },
     fieldNameFirst: { type: 'string' },
     // This isn't a node-office $ref because we only want
     // some of the properties in the entity
```